lib/krikri/harvester.rb

```diff
@@ -102,7 +102,12 @@ def get_record(_)
     # @return [Boolean]
     def run(activity_uri = nil)
       log :info, 'harvest is running'
-      records.each { |rec| rec.save(activity_uri) }
+      begin
```

Comment:
It's nice to have a string that we can reliably grep on, so your example above would be an improvement.